tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -15,36 +15,82 @@
  */
 package io.dockstore.tooltester.client.cli;
 
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+
+import com.offbytwo.jenkins.JenkinsServer;
+import com.offbytwo.jenkins.helper.JenkinsVersion;
+import com.offbytwo.jenkins.model.Job;
 import io.swagger.client.ApiClient;
 import io.swagger.client.ApiException;
 import io.swagger.client.Configuration;
 import io.swagger.client.api.ContainersApi;
 import io.swagger.client.api.GAGHApi;
 import io.swagger.client.api.UsersApi;
+import io.swagger.client.model.DockstoreTool;
+import io.swagger.client.model.SourceFile;
 import io.swagger.client.model.Tool;
+import io.swagger.client.model.ToolVersion;
 import joptsimple.OptionParser;
 import joptsimple.OptionSet;
 import org.apache.commons.configuration.ConfigurationException;
 import org.apache.commons.configuration.HierarchicalINIConfiguration;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.File;
-import java.util.List;
-import java.util.concurrent.atomic.AtomicBoolean;
-
 /**
  * Prototype for testing service
  */
 public class Client {
-
     private static final Logger LOG = LoggerFactory.getLogger(Client.class);
     private final OptionSet options;
     private ContainersApi containersApi;
     private UsersApi usersApi;
     private GAGHApi ga4ghApi;
     private boolean isAdmin = false;
+    private boolean development = false;
+    private Report report;
+
+    public DockerfileTester getDockerfileTester() {
+        return dockerfileTester;
+    }
 
+    public ParameterFileTester getParameterFileTester() {
+        return parameterFileTester;
+    }
+
+    private DockerfileTester dockerfileTester;
+    private ParameterFileTester parameterFileTester;
+
+    public JenkinsServer getJenkins() {
+        return jenkins;
+    }
+
+    private void setJenkins(JenkinsServer jenkins) {
+        this.jenkins = jenkins;
+    }
+
+    private JenkinsServer jenkins;
+
+    public void setCount(int count) {
+        this.count = count;
+    }
+
+    public int getCount() {
+        return count;
+    }
+
+    private int count = 0;
```

Comment:
please order your constants before instance variables, and instance variables before methods